swap/tests/alice_and_bob_refund_using_cancel_and_refund_command_timelock_not_expired.rs

```diff
@@ -38,13 +38,8 @@ async fn given_alice_and_bob_manually_cancel_when_timelock_not_expired_errors()
         ));
 
         // Bob tries but fails to manually cancel
-        let result = cli::cancel(bob_swap.id, bob_swap.bitcoin_wallet, bob_swap.db, false)
-            .await?
-            .unwrap_err();
-        assert!(matches!(
-            result,
-            cli::cancel::Error::CancelTimelockNotExpiredYet
-        ));
+        let result = cli::cancel(bob_swap.id, bob_swap.bitcoin_wallet, bob_swap.db).await;
+        assert!(result.is_err());
```

Comment:
Can we assert against the particular string? You should be able to downcast this to an Electrum protocol error.